app/components/cms_horizontal_card_component/cms_horizontal_card_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render GovGridRowComponent.new(padding: {top: 6, bottom: 0}) do |row|  %>
+  <%= row.with_column("full") do %>
+    <%= content_tag :div, class: [
+      "horizontal-card-component__wrapper",
+      @colour_theme ? cms_colour_theme(@colour_theme, "left") : nil].compact do %>
+
+      <div class="horizontal-card-component__content">
+        <h1 class="govuk-heading-m"><%= @title %></h1>
+        <%= render CmsRichTextBlockComponent.new(blocks: @body_blocks, with_wrapper: false) %>
+        <%= render @icon_block.render if @icon_block%>
+      </div>
+      <% if @image %>
+        <div class="horizontal-card-component__image">
+            <%= @image_link&.then { |link| link_to(link) { render CmsImageComponent.new(@image) } } || render(CmsImageComponent.new(@image)) %>
```

Comment:
Could we simplify this by having the link as part of the CmsImageComponent as an optional parameter?

As we have a few places where we might link an image, we could standardise it, rather than having to do it in lots of different components